src/Workspaces/CSharp/Portable/Extensions/MemberDeclarationSyntaxExtensions.cs

```diff
@@ -331,7 +331,10 @@ public static ArrowExpressionClauseSyntax GetExpressionBody(this MemberDeclarati
             switch (memberDeclaration?.Kind())
             {
                 case SyntaxKind.PropertyDeclaration:
-                    return ((PropertyDeclarationSyntax)memberDeclaration).ExpressionBody;
+                    var propertyDeclaration = (PropertyDeclarationSyntax)memberDeclaration;
+                    return propertyDeclaration.ExpressionBody ??
+                        propertyDeclaration.AccessorList.Accessors.FirstOrDefault(a => a.ExpressionBody != null)
```

Comment:
I don't really like this change - it seems like it's making this method lie and give the expression body of a random accessor, not the actual memberDeclaration given.

Feels like the calling code should be adjusted to deal with this case.